docs/site/home.html

```diff
@@ -96,26 +96,35 @@ <h3 class="block-title">What is it?</h3>
         </div>
 
         <div hljs language="js" ng-show="selected == 'compute engine'">
-// Auth is handled automatically inside GCE.
-var gcloud = require('gcloud');
-var dataset = gcloud.datastore.dataset();
-var productKey = dataset.key(['Product', 123]);
+// Authentication is automatic
+// inside Google Compute Engine.
+var gcloud = require('gcloud')({
+  projectId: 'grape-spaceship-123'
+});
+
+var gcs = gcloud.storage();
 
-dataset.get(productKey, function(err, entity) {
-  console.log(err, entity);
+var backups = gcs.bucket('backups');
+backups.upload('db.zip', function(err, file) {
```

Comment:
Just to be clear, this uploads the local file, right?

Wouldn't the logical next action be something logging that the upload worked or checking if there were errors?

```javascript
if (err) {
  pager.create(/* Wake up your dev team! */);
} else {
  log('Backup of ' + file + ' successful.');
}
```